frappe/desk/treeview.py

```diff
@@ -37,13 +37,17 @@ def get_all_nodes(doctype, label, parent, tree_method, **filters):
 
 
 @frappe.whitelist()
-def get_children(doctype, parent="", **filters):
-	return _get_children(doctype, parent)
+def get_children(doctype, parent="", include_disabled=False, **filters):
+	if isinstance(include_disabled, str):
+		include_disabled = bool(include_disabled)
```

Comment:
```suggestion
```
This explicit casting isn't really needed, the `if` check later will handle it

If we want to have an explicit boolean value passed anyway, can adjust this check for check for all non-bool types, instead of just converting strings.

Reply:
Realised the method should be different for correctly handling the type for the flag. Have added a change for that.

Tried testing it without explicitly typecasting the variable. Since this method is called from the client side, the arguments once come in as `string` type for getting the immediate children of the root. 

The reason for adding the condition to check if the argument is indeed a string was to avoid typecasting the variable when it already comes as a `boolean` value when recursion occurs (ie.  for subtrees).